spec/controllers/users_controller_spec.rb

```diff
@@ -127,7 +127,8 @@
       it 'fails to invite user when form params missing' do
         post :new, :params => { :team_id => 0 }
         expect(response).to have_http_status(:bad_request)
-        expect(flash.now[:errors]).not_to be_nil
+        expect(flash.now[:errors]).to be_nil
```

Comment:
I'm not sure I'd include this as it seems like resurrecting past history. It's not like there's a regression  bug that we're trying to test. But 🤷‍♂

Reply:
and removed this.  I left it in as it was a change to the existing test.